developer/local/postgres/db-statefulset.yaml

```diff
@@ -5,72 +5,61 @@ metadata:
   labels:
     app.kubernetes.io/name: tekton-results-database
     app.kubernetes.io/part-of: tekton-results
-    app.kubernetes.io/version: v0.4.0
+    app.kubernetes.io/version: devel
   name: tekton-results-database
   namespace: tekton-results
 spec:
   replicas: 1
   selector:
     matchLabels:
       app.kubernetes.io/name: tekton-results-database
-      app.kubernetes.io/part-of: tekton-results
-      app.kubernetes.io/version: v0.4.0
   serviceName: postgres
   template:
     metadata:
       labels:
         app.kubernetes.io/name: tekton-results-database
-        app.kubernetes.io/part-of: tekton-results
-        app.kubernetes.io/version: v0.4.0
+        app.kubernetes.io/version: devel
     spec:
       containers:
-        - envFrom:
-            - configMapRef:
-                name: tekton-results-database
-          image: postgres:13.3
-          name: postgres
+        - name: postgres
+          image: bitnami/postgresql@sha256:23b9a21460fefdd83accd0f864e734c88bebc67c86ee752a97b77dd4843907f0 # 13.3.0
           env:
-            - name: PGDATA
-              value: /var/lib/postgresql/data/pgdata
             - name: POSTGRES_USER
               valueFrom:
                 secretKeyRef:
-                  key: db.user
                   name: tekton-results-database
+                  key: db.user
             - name: POSTGRES_PASSWORD
               valueFrom:
                 secretKeyRef:
+                  name: tekton-results-database
                   key: db.password
+            - name: POSTGRES_DB
+              valueFrom:
+                secretKeyRef:
                   name: tekton-results-database
-          securityContext:
-            allowPrivilegeEscalation: false
-            capabilities:
-              drop:
-                - "ALL"
-            runAsNonRoot: true
-            seccompProfile:
-              type: RuntimeDefault
+                  key: db.name
           ports:
             - containerPort: 5432
               name: postgredb
           volumeMounts:
-            - mountPath: /var/lib/postgresql/data
-              name: postgredb
-            - mountPath: /docker-entrypoint-initdb.d
-              name: sql-initdb
-      volumes:
-        - configMap:
-            name: tekton-results-sql-initdb-config
-          name: sql-initdb
+            - name: postgredb
+              mountPath: /bitnami/postgresql
+          securityContext:
+            seccompProfile:
+              type: RuntimeDefault
+            runAsNonRoot: true
+            allowPrivilegeEscalation: false
+            capabilities:
+              drop:
+                - ALL
+              add:
+                - NET_BIND_SERVICE
```

Comment:
I don't even think we need this capability. This lets the container bind to ports lower than 1024.

See https://man7.org/linux/man-pages/man7/capabilities.7.html

Reply:
removed